merge-and-build.sh

```diff
@@ -76,11 +76,11 @@ git merge master -m "Merge master into enterprise-${ose_version}"
 cd "${workpath}"
 git clone git@github.com:openshift/ose.git
 cd ose
-git add remote upstream git@github.com:openshift/origin.git
-git fetch --all
+git remote add upstream git@github.com:openshift/origin.git
+git fetch upstream master
```

Comment:
This should speed things up by avoiding fetching anything from ose (which we just cloned) and other refs from origin.